docs/ncu-ci.md

```diff
@@ -226,6 +227,40 @@ Notifying upstream projects of job completion
 Finished: SUCCESS
 ```
 
+### `ncu-ci citgm <jobid>`
+
+`ncu-ci citgm <jobid>` shows the results of a given citgm-smoker job.
+
+Example:
+```
+node on git:master ❯ ncu-ci citgm 2400                                         10:25AM
+--------------------------------------------------------------------------------
+[1/1] Running CITGM: 2400
+--------------------------------------------------------------------------------
+✔  Header data downloaded
+✔  Report data downloaded
+----------------------------------- Summary ------------------------------------
+Result     FAILURE
+URL        https://ci.nodejs.org/job/citgm-smoker/2400/testReport/
+Source     https://github.com/nodejs/node/pull/34093/
+Commit     [9ec07f42864c] 2020-06-30, Version 14.5.0 (Current)
+Date       2020-06-29 21:17:56 -0700
+Author     Shelley Vohr <shelley.vohr@gmail.com>
+----------------------------------- Failures -----------------------------------
+┌────────────────────────┬───────────────────────┬───────────────────────┬─────────────────────────┬─────────────────────┬─────────────────┬────────────────────┐
+│        (index)         │           0           │           1           │            2            │          3          │        4        │         5          │
```

Comment:
The numbers on this row represent how many tests failed, correct? Might be worth explaining it somewhere.

Reply:
o great point! i grab those numbers so we can definitely print them out helpfully 🤔

Reply:
Added a statistics table and also added that selfsame info to the markdown and json outputs (updated in PR body)